app/controllers/search_controller.rb

```diff
@@ -1,11 +1,13 @@
 class SearchController < ApplicationController
   layout "search_layout"
-  before_action :set_expiry
 
   def index
     search_params = SearchParameters.new(params)
 
-    @content_item = Services.cached_content_item("/search")
+    content_item = Services.cached_content_item("/search")
+    @content_item = content_item.to_h
```

Comment:
Is the `content_item` already a hash here?

Reply:
yes, I've updated